app/react/Connections/actions/actions.js

```diff
@@ -1,22 +1,26 @@
+import qs from 'qs';
```

Comment:
I think we already had querystring as a depencency? Is this the same thing?

Reply:
@RafaPolit [not the same thing](https://stackoverflow.com/questions/29175465/body-parser-extended-option-qs-vs-querystring/29177740#:~:text=The%20main%20advantage%20of%20qs,any%20json%2Dlike%20data%20structure.&text=querystring%20library%60%20provides%20basic%20serialization,browsers%20to%20serialize%20form%20data.) i have used qs mainly because of its ability to parse nested objects.
i have also removed query-string as a dependency and changed the previous code to use qs instead.